lib/libutils/isoc/include/stdlib.h

```diff
@@ -29,4 +29,14 @@ int rand(void);
 unsigned long _strtoul (const char *nptr, char **endptr, int base);
 unsigned long strtoul (const char *s, char **ptr, int base);
 
-#endif /* STDLIB_H */
+#if __STDC_VERSION__ >= 201112L
+void *aligned_alloc(size_t alignment, size_t size);
```

Comment:
I looked at the man pages, everything is supposed to be made available via `<stdlib.h>` except `memalign()` which requires `<malloc.h>` (and AFAICT should not be pulled by `<stdlib.h>` alone). Our `<stdlib.h>` includes `<malloc.h>`, so it doesn't strictly comply regarding `memalign()`. glibc and newlib declare `calloc()`, `malloc()` and `realloc()` in  `<stdlib.h>` AND `<malloc.h>` (yes the prototypes are present twice). `aligned_alloc()` however is only in `<stdlib.h>`. So if we want `aligned_alloc()` NOT to be available ONLY when `<stdlib.h>`, it belongs to `<stdlib.h>`. 

Now I realize this explanation is quite convoluted and perhaps we should not care too much about this level of detail. Availability (or rather non-availability) of function should be based on `-sdt=xxx` and macros rather than which header exactly is included I suppose. So I won't object if you prefer to group `aligned_alloc()` with the others in `malloc.h`.